src/ecalc/libraries/libecalc/common/libecalc/core/consumers/compressor/component.py

```diff
@@ -94,10 +98,53 @@ def evaluate(
                 values=aggregated_result.is_valid, timesteps=evaluated_timesteps, unit=Unit.NONE
             ),
             id=self.id,
+            recirculation_loss=TimeSeriesRate(
+                values=aggregated_result.recirculation_loss,
```

Comment:
What about regularity here? Should it not be included in all timeseries?

Reply:
Updated :)

Not sure exactly where we want to isolate regularity. I think it is Okay to pass it down to the consumer, but we want to investigate if we can remove it completely from libecalc.core